modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+NOTE: Enablng this feature depends on changes made to the Core {productname} editor. As a consequence, showing avatars next to the user-name in comments requires both {productname} 6.1 or later and version 3.1.0 or later of the xref:comments.adoc[Comments Premium Plugin].
+
+
+==== Show comments sidebar on editor initialization
+
+It is now possible to initialize a {productname} editor with a comment sidebar automatically visible and available but with focus still in the main editor.
+
+To add a comment sidebar add `sidebar_show: 'showcomments'` to `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+    selector: "textarea",
+	sidebar_show: 'showcomments',
+});
+----
+
+For a customised comments sidebar, setup the sidebar in the `setup` section of `tinymce.init`:
+
+[source,js]
+----
+	setup: (editor) => {
+	  editor.ui.registry.addSidebar('showcomments', {
+		tooltip: 'showcomments',
+		icon: 'comment',
+		onSetup: (api) => {
+		  console.log('Render panel', api.element());
+		},
+		onShow: (api) => {
+		  console.log('Show panel', api.element());
+		},
+		onHide: (api) => {
+		  console.log('Hide panel', api.element());
+		}
+	  });
+	}
+----
+
+NOTE: The comment sidebar presents with no animation or other effect. It is just there, along with any other initialised {productname} editor components. Also, as noted above, the sidebar does not take focus, which remains with the main editor.
+
+
+==== Support annotating certain block elements directly when using the editor annotator API
+
+Several new block elements can now have comments attached to them, including:
+
+* Tables of contents
+* Images
+* Images with captions
+* Code samples
+* media in iframes
+* embedded media
+* embedded pages
+
+When comments are showing, a 3px wide yellow (hex color #FFE89D) rectangle presents around any element with a comment attached.
+
+NOTE: Enablng this feature depends on changes made to the Core {productname} editor. As a consequence, adding comments to these elements requires both {productname} 6.1 or later and version 3.1.0 or later of the xref:comments.adoc[Comments Premium Plugin].
+
+
+==== The label name for the Save Comment button can now be customized
+
+Previously, the Save Comment was labeled *Save*. After field reports that end-users were interpreting this to mean the entire {productname} document would be saved on pressing this button, two changes have been made.
+
+1. The default label for this button is now *Comment*.
++
+This new default presents for both new comments and replies to existing comments.
+
+2. This default label is now customisable.
++
+_How does one customise this? I presume it’s a value one sets as part of `tinymce.init` but which one? And where in this configuration file?_
++
+_Or is it a matter of an integrator editing the `.pot` file — `src/text/pot/plugins/tinycomments/tinymce_tinycomments.pot` — that comes along for the ride?_
```

Comment:
I believe the 'tinycomments_js_url' option can be used to override the default implementation, though there is no documentation for it so I can't confirm anything

Reply:
The option takes a string, with the default as `pluginUrl + '/js'`, and is used as the directory in retrieving 'tinycomments-sidebar.min.js'. I believe this can be overridden with the location of a custom 'tinycomments-sidebar.min.js' containing customised UI labels.

Reply:
Or perhaps the value can just be updated dynamically within the webpage

Reply:
Customisation of strings is done using `addI18n`, which is a global method shared by all tinymce instances on the page.
https://www.tiny.cloud/docs/tinymce/6/apis/tinymce.root/#addI18n

```
tinymce.addI18n('sv_SE', {
   'Comment': 'Spara kommentar'
});
```

Reply:
It's also not really customization... it's just providing a custom translation. I personally don't think we should be including this is release notes, it's just something we tell if someone logs a support ticket normally.

Re `tinycomments_js_url` that's an internal option so shouldn't be mentioned in docs.

Reply:
To quote my `git` commit message:

edits to Save Comment button section.

1. Title edited to reflect it is about a new default and translatability, not customisation.

2. added code example of how to translate the button.